benches/benches/bevy_ecs/schedule.rs

```diff
@@ -0,0 +1,62 @@
+use bevy_app::App;
+use bevy_ecs::prelude::*;
+use criterion::{criterion_group, criterion_main, Criterion};
+
+criterion_group!(benches, build_schedule);
+criterion_main!(benches);
+
+fn build_schedule(criterion: &mut Criterion) {
+    // empty system
+    fn sys() {}
```

Comment:
I think we should use non-empty systems here with somewhat diverse data access patterns here. One of the more important optimizations in this area of code is to immediately optimize-away edges between systems with compatible data accesses.

These systems will all be compatible, and so there's no way to spot that.

Hmm, maybe this is a good choice for another dimension to split on?

Reply:
I don't think we want to add data conflicts or we'll start testing the scheduler too, instead of just the initialization.

We could maybe add a single threaded stage and add the systems to that. That would probably give the most consistent results.

Reply:
Yeah fair. I could go either way on the single-threaded stage; my personal feeling is that this is good to merge.